src/config.ts

```diff
@@ -100,6 +100,10 @@ export const LIQUIDITY_POOL_REGISTRY_ADDRESS: string | undefined = _.isEmpty(
           EnvVarType.ETHAddressHex,
       );
 
+export const RFQT_API_KEY_WHITELIST: string[] = (process.env.RFQT_API_KEY_WHITELIST || '[]').split(',');
+
+export const RFQT_MAKER_ENDPOINTS: string[] = (process.env.RFQT_MAKER_ENDPOINTS || '[]').split(',');
```

Comment:
Same comment as above

Reply:
Addressed in https://github.com/0xProject/0x-api/pull/162/commits/e1693bba254b24852fbedae88b25bf8ebedc5616